rules-tests/DowngradePhp70/Rector/Spaceship/DowngradeSpaceshipRector/Fixture/echo.php.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\Coalesce\DowngradeStrictTypeDeclarationRector\Fixture;
+
+class UseEcho
+{
+    public function run($a, $b)
+    {
+        echo $a <=> $b;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\Coalesce\DowngradeStrictTypeDeclarationRector\Fixture;
+
+class UseEcho
+{
+    public function run($a, $b)
+    {
+        echo (function ($a, $b) {
+            if ($a === $b) {
+                return 0;
+            }
+            return $a < $b ? -1 : 1;
+        })($a, $b);
```

Comment:
While this might be technically correct, I find it very hard to read - definition vs call. Similar to class vs anonymous class.

Could you use an anonymous function assinged to a variable? 

```diff
+$battleShipcompare = function ($a, $b) {
-echo (function ($a, $b) {
            if ($a === $b) {
                return 0;
            }
            return $a < $b ? -1 : 1;
-        })($a, $b);
+        });
+$battleShipcompare($a, $b);
```

Reply:
updated to assign to variable `battleShipcompare` first at https://github.com/rectorphp/rector/pull/6243/commits/cb6e1e46bb376592f26e2c3caaa42169744265a3